### ISSUE TITLE
Aspia host on linux

### ISSUE THREAD

**Issue by ddkmoonster** [dchapyshev/aspia#173]
Привет! У тебя очень интересный проект, вполне завершенный. Однако очень расстроило отсутствие Host версии для Linux.
Я являюсь системным администратором, , очень много серверных решений для IP телефонии и прочего прочего, что использует эту ОС. Я не являюсь ее фанатом, но реалии таковы, что без Linux никуда.
Пожалуйста, прошу, сосредоточься на разработке host на Linux.

**Comment by ProtsenkoAV** [comment]
Да, хост под линукс нужен. В рамках импортозамещения уже много не только серверов, но и рабочих станций под линуксом :(

**Comment by extreen** [comment]
Очень необходима версия под линукс.
Присоединяюсь к просьбе сосредоточиться на разработке host на Linux.

**Comment by DragonZX** [comment]
Присоединяюсь к просьбе, хочу заменить последние лицензии удаленного управления линукс-десктопами

**Comment by sivalexei** [comment]
Присоединяюсь к просьбе сосредоточиться на разработке host на Linux

**Comment by antonvladimirov** [comment]
Поддерживаю!

**Comment by Babikov-gigabit** [comment]
Скомпилировал версию для debian 12 (Astra linux), может будет кому полезно. Буду благодарен если автор проекта сказал если я что то упустил или сделал не так [ https://github.com/Babikov-gigabit/Aspia_HOST_Linux ]
